mod/PackageDefinitions/paulwalkeruk-tiger-moth-aerobatic-smoke.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<AssetPackage Version="1.0.0">
+<AssetPackage Version="2.0.0">
```

Comment:
I don't see this next release will contain any breaking changes, so I'd like the next version to be 1.1.0 (following the SemVer approach)
```suggestion
<AssetPackage Version="1.1.0">
```